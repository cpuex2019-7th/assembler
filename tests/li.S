li a0, 0x7fffffff
