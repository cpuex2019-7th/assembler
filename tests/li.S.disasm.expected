lui a0, 524288
addi a0, a0, -1
